src/databricks/labs/ucx/source_code/linters/imports.py

```diff
@@ -116,6 +116,11 @@ def __init__(self, session_state: CurrentSessionState):
         self._session_state = session_state
 
     def lint(self, code: str) -> Iterable[Advice]:
+        # can't find a way to avoid this cyclic import
+        # pylint: disable=import-outside-toplevel, cyclic-import
+        from databricks.labs.ucx.source_code.notebooks.magic import MagicCommand
```

Comment:
Disabling cyclic import checker is not allowed

Reply:
done